frameworks/sveltekit/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "kit-prisma",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "svelte-kit dev",
+    "build": "svelte-kit build && cp node_modules/prisma/*query* .vercel_build_output/functions/node/render/;cp prisma/schema.prisma .vercel_build_output/functions/node/render/",
+    "package": "svelte-kit package",
+    "preview": "svelte-kit preview",
+    "prepare": "svelte-kit sync",
+    "check": "svelte-check --tsconfig ./tsconfig.json",
+    "check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
+    "vercel-build": "prisma generate && prisma migrate deploy && yarn build"
```

Comment:
Same as https://github.com/prisma/ecosystem-tests/pull/2695/files#r866773654 for this one